### PR TITLE
Fix various minor issues

### PR DESCRIPTION
**Pull request by mmmavis**
## Related PRs/issues:

Review app login: https://foundation-s-fix-minor--meywng.herokuapp.com/cms/ (credential: `admin2/admin2password`)

### #9666 Enlarged nav dropdown's vertical gap to 16px
**Commit**: https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11960/commits/2e9c5995b43649732537db5bc16d5348a847f06e
**Test page**: https://foundation-s-fix-minor--meywng.mofostaging.net/en/who-we-are/

### #10857 Adjusted PNI category dropdown button font size
**Commit**: https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11960/commits/b8b8de3e4eff0bb79ce56d2653eb37b74c8dde61
**Test page**: https://foundation-s-fix-minor--meywng.mofostaging.net/en/privacynotincluded/

### #9941 Capitalized all instances of lowercase *PNI
**Commit**: https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11960/commits/17f1f19c5019f94c1477bdb39cc854ae6259a816
**Test page**: https://foundation-s-fix-minor--meywng.mofostaging.net/en/privacynotincluded/
**Note for code reviewer**: Commit includes a migration file due to updating a `verbose_name` but I'm debating if this change is really worth having a migration file for or we just leave this particular instance of lowercase *PNI as is.

### #10121 Made spacer available on ArticlePage
**Commits**: https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11960/commits/ddccad160c4bf7b0ac7388bb6a1afbe3e6a8b5b7 and https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11960/commits/fea35ba57485b0d96748480bfe4b077df5791cd5
**Test page**: https://foundation-s-fix-minor--meywng.herokuapp.com/cms/pages/127/edit/ (credential: `admin2/admin2password`)

### CODE REVIEW

File: network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
                tw-mr-auto
                tw-font-sans
                tw-font-bold
-               tw-text-sm
+               tw-text-[15px]
```
Comment (jhonatan-lopes) [inline network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html:13]: Although `.pni-category-nav a` defines a 15px font size, I'm thinking if it wouldn't be better to change it to 16px. This way we could use `tw-text-base` here (which is 16px) and avoid the one-off style...

**Review by jhonatan-lopes — COMMENTED** [review]
Hey @mmmavis, I just have a minor observation

File: network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html (hunk 1 above)
Comment (mmmavis) [reply network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html:13]: Hey @jhonatan-lopes , good suggestion! I thought about it but then recalled there are a bunch of PNI existing styles that need to be updated to adapt to our design system. For now I'm thinking that we just make them consistent across the PNI pages and clean them up altogether as a later phase.

**Review by nancyt1 — CHANGES_REQUESTED** [review]
For #9941, the ding icon on product pages (#4) is still showing up lowercase. We could also pull this out into another ticket if it's too much work since it's relatively minor. Other than that, everything else looks great!

File: network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html (hunk 1 above)
Comment (jhonatan-lopes) [reply network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html:13]: Sounds good

**Review by jhonatan-lopes — APPROVED** [review]

**Comment by mmmavis** [comment]
> For #9941, the ding icon on product pages (#4) is still showing up lowercase. We could also pull this out into another ticket if it's too much work since it's relatively minor. Other than that, everything else looks great!

@nancyt1 I just took a look but couldn't reproduce the issue you found. Do you mind checking again?

image

**Comment by nancyt1** [comment]
@mmmavis 
It's this icon that still uses lowercase. 

SCR-20240305-ouqj

**Comment by mmmavis** [comment]
@nancyt1 Ah I see. Thanks for pointing that out. In that case could I ask you to help update [this image](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/main/source/images/buyers-guide/icon-privacy-ding.svg) with upper case text? If it takes too much effort, I can file a new ticket for us to tackle later

**Comment by nancyt1** [comment]
@mmmavis Here's the [folder](https://drive.google.com/drive/u/0/folders/1XpTWS3vP0taMh9WCP-U9NXCcaMKfUMfr) of the SVGs! It also has localized versions.

**Comment by mmmavis** [comment]
@nancyt1 Thanks! And sorry I didn't realize you included the link in the initial ticket already. Time to get some coffee Mavis ☕

**Comment by mmmavis** [comment]
So that we can have these old tickets addressed and closed sooner, I've replaced the current ding images with the new ones. In the future, we should move towards excluding text from images.

@nancyt1 Please verify the changes here: https://foundation-s-fix-minor--meywng.herokuapp.com/en/privacynotincluded/gas-finish-hit-speak-eye-store/ Thanks!

**Comment by nancyt1** [comment]
@mmmavis hm, they still show up as lowercase for me. Can you screenshot what you see?

**Comment by mmmavis** [comment]
@nancyt1 sorry, I updated everything except the English version. This should be fixed now 😅

**Review by nancyt1 — APPROVED** [review]
All looks good! (:

**Pull request merged**